packages/components/src/internal/components/settings/NameIdSettings.tsx

```diff
@@ -233,13 +328,133 @@ export const NameIdSettingsForm: FC<NameIdSettingsFormProps> = props => {
                     </div>
                 )}
 
+                <div className="sample-counter__setting-section">
+                    <div className="list__bold-text margin-bottom">Naming Pattern Elements/Tokens</div>
+                    <div>
+                        The following tokens/counters are utilized in naming patterns for the project and all
+                        sub-projects. To modify a counter, simply enter a number greater than the current value and
+                        click “Apply”. Please be aware that once a counter is changed, the action cannot be reversed.
+                        For additional information regarding these tokens, you can refer to this{' '}
+                        <HelpLink topic={SAMPLE_TYPE_NAME_EXPRESSION_TOPIC}>link</HelpLink>.
+                    </div>
+
+                    {loading && <LoadingSpinner />}
+                    {!loading && (
+                        <>
```

Comment:
It looks like this `<>` had already been changed to a `<div>` but I don't think this was the type of case that would have caused issues. It seems to be just `<>` fragments that have a string as the contents and not components. So this can likely be changed back to a fragment if you'd like @XingY .